pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceTest.java

```diff
@@ -1674,6 +1676,18 @@ public void testMetricsPersistentTopicLoadFails() throws Exception {
         admin.topics().createNonPartitionedTopic(topic);
         admin.topics().unload(topic);
 
+        // Get original counter.
+        MutableInt failedLoadTopic1 = new MutableInt(0);
+        MutableInt concurrencyLoadTopicAndUnloadBundle1 = new MutableInt(0);
```

Comment:
since multiple threads are involved, shouldn't these be AtomicIntegers?

Reply:
I don't currently understand the reason to use `MutableInt`s. 
If the reason is to that a final variable is needed for an inner lambda, it might be better to use a final variable.
One trick that could be useful is to isolate variable scopes with a `{....}` block. That would allow to introduce the same variable name multiple times in a single unit test without variable name collisions since the variables would e in different scopes.

Reply:
Fixed